com.github.halfmexican.Mingle.json

```diff
@@ -0,0 +1,59 @@
+{
+    "app-id" : "com.github.halfmexican.Mingle",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "46",
+    "sdk" : "org.gnome.Sdk",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.vala"
```

Comment:
I don't think this is necessary. The GNOME sdk contains Vala.

Reply:
removed